src/Xamarin.Android.Tools.AndroidSdk/Tests/AndroidSdkInfoTests.cs

```diff
@@ -15,5 +15,18 @@ public void Constructor_NullLogger ()
 			Action<TraceLevel, string> logger = null;
 			Assert.Throws<ArgumentNullException> (() => new AndroidSdkInfo (logger));
 		}
+
+		[Test]
+		public void AndroidSdkAndNdk_Exist ()
```

Comment:
I'm not sure how I feel about this test. All it's really asserting is that *one* of the "well known" Android SDK directories exists on the machine running the unit tests.  This doesn't actually let us know that the `vs_2017_default` location added as part of this patch is one of them...nor could it. (How could it?)

Additionally, asserting that the NDK path exists is wrong, as the NDK is *optional*; see the `AndroidSdkInfo` constructor, in which we only assert that the Android SDK and Java JDK paths are valid.